js/src/components/google-combo-account-card/claim-ads-account/claim-ads-account.scss

```
@@ -0,0 +1,30 @@
+.gla-claim-ads-account-section {
+	padding: 0;
+
+	.gla-claim-ads-account-box {
+		border: 1px solid $gray-300;
+		display: flex;
+		flex-direction: column;
+		gap: $grid-unit-10;
+		padding: $grid-unit-30;
+
+		h4 {
+			font-size: $gla-font-small;
+			margin: 0;
+		}
+
+		p {
+			margin: 0;
+		}
+
+		.gla-ads-post-claim-instructions {
+			color: $gray-700;
+			font-style: italic;
+		}
+
+		.app-button {
+			display: flex;
+			margin-left: auto;
```

After the parent was changed to flex layout, it could be aligned by:

```suggestion
			align-self: flex-end;
```

---

Nice catch! Updated!